README.md

```diff
@@ -14,29 +14,48 @@ A set of [React](https://reactjs.org/) components created by, and for, [Mavenlin
   yarn add @mavenlink/design-system
   ```
 
-- Setup [CSS modules](https://github.com/css-modules/css-modules). One way of doing that is with [css-loader for Webpack](https://github.com/webpack-contrib/css-loader#modules)
+- Setup [React JSX processing](https://reactjs.org/docs/jsx-in-depth.html). One way of doing that is with [`babel-loader` for Webpack](https://webpack.js.org/loaders/babel-loader/).
+- Setup [CSS modules](https://github.com/css-modules/css-modules). One way of doing that is with [`style-loader` for Webpack](https://github.com/webpack-contrib/style-loader) and [`css-loader` for Webpack](https://github.com/webpack-contrib/css-loader).
+- Setup [SVG processing](https://svgontheweb.com/#spriting). One way of doing that is with [`svg-sprite-loader` for Webpack](https://github.com/kisenka/svg-sprite-loader).
 
   ```js
   // Webpack configuration
+  const SpriteLoaderPlugin = require('svg-sprite-loader/plugin');
+
   module.exports = {
     module: {
-      rules: [
-        {
-          test: /\.css$/,
+      rules: [{
```

Comment:
We have a Webpack file for our styleguidist configuration. It, however, has some problems because of styleguidist. This snippet of Webpack configuration makes it easier to pick up MDS as a dependency in a Webpack project without any mess.